README.md

```diff
@@ -70,8 +70,7 @@ You can run your application in dev mode that enables live coding using:
 ```shell script
 docker run -d --rm -e CLUSTER_NAME=dse-cluster -e CLUSTER_VERSION=6.8 -e ENABLE_AUTH=true -e DEVELOPER_MODE=true -e DS_LICENSE=accept -e DSE=true -p 8081:8081 -p 8091:8091 -p 9042:9042 stargateio/coordinator-dse-next:v2
 
-./mvnw compile quarkus:dev -Dstargate.data-store.ignore-bridge=true \
-  -Dstargate.jsonapi.operations.vectorize-enabled=true \
+./mvnw compile quarkus:dev -Dstargate.jsonapi.operations.vectorize-enabled=true \
```

Comment:
I know it's not part of cleanup, but should we add

    -Dstargate.feature.flags.TABLES=true

since it is needed currently.

Reply:
Thanks, maybe we can add when table feature is previewed. 
Will merge the PR.